scratch2/draw.js

```diff
@@ -59,11 +59,11 @@ var SVG = (module.exports = {
     return SVG.withChildren(SVG.el("g"), children)
   },
 
-  newSVG(width, height) {
+  newSVG(width, height, scale) {
     return SVG.el("svg", {
       version: "1.1",
-      width: width,
-      height: height,
+      width: width * scale,
+      height: height * scale,
       viewBox: `0 0 ${width} ${height}`,
```

Comment:
I think this is right, but do you happen to know of any documentation confirming that we can scale an SVG by setting the width/height and the `viewBox` to different values?

I’m just wondering if we should be setting them to the same thing, and then using an explicit `transform()` instead.

Reply:
In my understanding the two things are independent. I like to think of width and height as the "external" size of the image from the perspective of the website and viewBox as the "internal" size from the perspective of the image elements.

MDN doesn't indicate that they should be tied together (https://developer.mozilla.org/en-US/docs/Web/SVG/Element/svg), and it would be redundant if that would be the case.